pkg/config/remote/uptane/client.go

```diff
@@ -153,6 +158,25 @@ func (c *Client) TargetFile(path string) ([]byte, error) {
 	return buffer.Bytes(), nil
 }
 
+// TargetsMeta returns the current raw targets.json meta of this uptane client
+func (c *Client) TargetsMeta() ([]byte, error) {
+	c.Lock()
+	defer c.Unlock()
+	err := c.verify()
```

Comment:
Why call c.verify() in DirectorRoot, Targets, TargetMeta while the verification is done at Update time?

Reply:
The idea is to make reasoning about security straightforward if those methods somehow get called after a failed update. This is a failsafe to ensure nothing can get extracted of the client if the local state is not valid.